lib/config-generator.js

```diff
@@ -9,6 +9,7 @@
 
 'use strict';
 
+const WebpackConfig = require('./WebpackConfig'); //eslint-disable-line no-unused-vars
```

Comment:
I didn't realize we needed to import the class in order for the jsdoc to function below. If that's the case, I'm surprised eslint isn't smart enough to know that this really is "used"

Reply:
well, everything is scoped in JS. As our class is not a global variable, you cannot use it globally.